package.json

```diff
@@ -70,13 +70,18 @@
     "@storybook/addon-actions": "4.1.18",
     "@storybook/addon-storyshots": "4.1.18",
     "@storybook/react-native": "4.1.18",
+    "@tsconfig/react-native": "2.0.3",
+    "@types/jest": "29.4.0",
+    "@types/react": "18.0.28",
+    "@types/react-native": "0.71.3",
+    "@types/react-test-renderer": "18.0.0",
     "babel-eslint": "10.1.0",
     "babel-jest": "26.6.3",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-preset-react-native": "4.0.1",
     "chokidar": "3.5.1",
     "eslint": "7.20.0",
-    "eslint-plugin-instawork": "0.10.2",
+    "eslint-plugin-instawork": "https://github.com/Instawork/eslint-plugin-instawork#ashwinibm/feat-typescript-plugin",
```

Comment:
Will be updated once the [PR](https://github.com/Instawork/eslint-plugin-instawork/pull/41/) on this repo is merged.

Reply:
This is done.